src/App.scss

```diff
@@ -0,0 +1,301 @@
+@font-face {
+  font-family: Montserrat;
+  src: url('./fonts/Montserrat.ttf');
+}
+
+@font-face {
+  font-family: Montserrat-Light;
+  src: url('./fonts/Montserrat-Light.ttf');
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url('./fonts/Roboto.ttf');
+}
+
+$c1: #0290ff;
+$c2: #e8e8e8;
+$c3: #121212;
+$c4: #4386bf;
+
+body {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  margin: 0;
+  width: 100%;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+  border-bottom: solid 1px rgba(192, 192, 192, 0.301);
+  background-color: white;
+  padding-left: 7.5%;
+  padding-right: 7.5%;
+  width: 85%;
+  margin-left: auto;
+  margin-right: auto;
+  position: fixed;
+  top: 0;
+  z-index: 1;
```

Comment:
- [ ] Following CSS best Practices, it would be better to use shorthand properties. Instead of using margin-top, margin-right, margin-bottom, and margin-left separately you can simply use just one line: `margin: 20px auto 0 auto;`

Reply:
Thanks for your useful advice. It's done.